helm_deploy/hmpps-book-video-link/templates/_envs.tpl

```diff
@@ -34,12 +34,6 @@ env:
         name: {{ template "app.name" . }}
         key: APPINSIGHTS_INSTRUMENTATIONKEY
 
-  - name: GOOGLE_ANALYTICS_ID
```

Comment:
This is linked up in Tag Manager now, so no need to reference it directly in the app